netlify.toml

```diff
@@ -437,6 +437,128 @@
   status = 301
   force = true
 
+# Cloud Sidebar IA
+
+[[redirects]]
```

Comment:
Where are we anticipating these redirecting to be needed? I would expect the onlinks to update when this merges.

Reply:
gottchya - I imagine this will end up pretty unmanagable over time 😅

Reply:
Yes. Not sure there's a good fix for that 😐